.gitignore

```diff
@@ -1,7 +1,6 @@
 # compiled output
 /dist
 /node_modules
-.env
```

Comment:
Re add this because the .env never should be uploaded with the others files, instead this you should create a new file named ".env.example" with fake values